src/components/ServiceQuestionDetails.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import ServiceQuestionsService from "../services/ServiceQuestionsService";
+
+class ServiceQuestionDetails extends React.Component {
+    constructor(props) {
+        super(props);
+        this.serviceQuestionsService = ServiceQuestionsService.getInstance();
+        this.state = {
+            serviceQuestion: {
+                choiceAnswer: 'Placeholder Choice',
+                title: 'Placeholder Title',
+                question: 'Placeholder questions',
+                id: 1
+            }
+        }
+    }
+    componentDidMount() {
+        this.findById(this.props.id);

```

Comment:
I can't figure out how to load any record other than the default id 1. I've tried to put it in the url while loading and typing it in the box, but it doesn't seem possible? I'm not sure if I'm doing something wrong. I would do this in the same way that we did the FAQ Details and load all the service questions and let the user pick from a drop down which one they would like.